routers/user_router.py

```diff
@@ -34,3 +35,16 @@ def create_user(user: user_schema.UserCreate, db: Session = Depends(get_db)) ->
     db.refresh(new_user)
     return new_user
 
+@router.delete("/users/{user_email}", status_code=status.HTTP_204_NO_CONTENT)
+def delete_user(user_email: str, db: Session = Depends(get_db)) -> None: 
```

Comment:
The return type here is not none since you are returning "{f"user with email{user_email}, was successfully deleted"}"